src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,12 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo
+        key={post.id}
+        post={post}
+      />
+    ))}
+  </div>
```

Comment:
It is considered good practice to add a return statement to your function components for clarity, even though they may work without it.